packages/typography/index.demo.tsx

```diff
@@ -0,0 +1,36 @@
+import { h, Component } from 'skatejs';
```

Comment:
yeah I meant single demo component with all examples inside, not multiple demo components within single file if that is what you mean